### PR TITLE
MTO changelog

### PR DESCRIPTION
**Pull request by ShahidHussain-srti**
Add changelog for MTO
Add missing MTO hibernation docs
Fix some formatting issues

### CODE REVIEW

File: docs/content/sre/multi-tenant-operator/changelog.md [hunk 1]
```diff
@@ -1,26 +1,51 @@
...
 
+## v0.8.x
+
+**v0.8.1**
+
+- fix: Updated release pipelines
+
+**v0.8.0**
+
+- feat: Allow custom roles for each tenant via label selector, more details in [custom roles document](./usecases/custom-roles.md)
```
Comment (MahnoorAsghar) [inline docs/content/sre/multi-tenant-operator/changelog.md:11]: Add fix: Fix panic in ResourceSupervisor sleep functionality due to sending on closed channel

**Comment by stakater-user** [comment]
@LilShah Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-106-cc0be321`

**Comment by stakater-user** [comment]
@LilShah Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-106-073c86a3`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-106-f5101f1a`

**Review by rasheedamir — APPROVED** [review]

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.saap-prod.clusters.stakater.io/stakater/stakaterclouddocs:v0.0.77`

**Pull request merged**